assets/css/main.scss

```diff
@@ -384,6 +384,7 @@ $sidebar-display: "sidebar-display";
 }
 
 #search-results {
+  word-break: break-word;
```

Comment:
`break-word` will reduce the reading experience, don't use it.

Add `max-with: 100%;` between line 399 to 400 is the better approche

Reply:
Good point! 👍